projects/ngx-datatable/src/lib/components/body/body-row-def.component.ts

```diff
@@ -81,4 +83,6 @@ interface RowDefContext {
   rowTemplate: TemplateRef<unknown>;
   row: any;
   index: number;
+  rowWrapper: DataTableRowWrapperComponent;
+  group: RowOrGroup<unknown>;
```

Comment:
this is affecting public API. At least `DataTableRowWrapperComponent` should no be exposed.

I think creating a new interface here would be better

Reply:
removed `rowWrapper` but I am keeping `group` as it make sense to have it here.